scale-ui/app/modules/jobs/partials/jobDetailTemplate.html

```diff
@@ -27,11 +27,20 @@ <h4 class="panel-title error-title"><strong>{{ vm.job.error.category }} Error: {
                         {{ vm.job.error.description }}
                     </div>
                 </div>
+                <div class="panel panel-info" ng-show="vm.job.is_superseded">
+                    <div class="panel-heading">
+                        <div class="panel-title">
+                            <strong>SUPERSEDED {{ vm.supersededChange }}</strong>
+                        </div>
+                    </div>
+                </div>
```

Comment:
Can you add the text "This job is obsolete because its recipe has been reprocessed." to the superseded box?
I would like to communicate to the user what SUPERSEDED means.

Also I think the SUPERSEDED box might be better above the status box rather than below it. If a job is superseded, its status seems like secondary importance.